apps/angular/src/core/services/url-config.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
+
+/** Service for setup url config for requests to API. */
+@Injectable({ providedIn: 'root' })
+export class UrlConfigService {
+
+	/** Object with parts of the url requests. */
+	public readonly endpoints = {
+		anime: 'anime/anime/',
+	};
```

Comment:
The name of the object means that all endpoints will be contained here. But for a large number of endpoints, this is inconvenient. It is better to create an object for each endpoint group. For example, now we have an `anime` group, but we can have an `auth` group to log in and register.

```suggestion
	public readonly anime = {
		getList: 'anime/anime/',
	};
```